pmdarima/__init__.py

```diff
@@ -19,10 +20,13 @@
 #   X.YrcN  # Release Candidate
 #   X.Y     # Final release
 #
-# Dev branch marker is: 'X.Y.dev' or 'X.Y.devN' where N is an integer.
-# 'X.Y.dev0' is the canonical version of 'X.Y.dev'
-#
-__version__ = "1.4.1.dev0"
+# We only create a VERSION file in CI/CD on tagged commits.
+# For local development, or non-tagged commits, we will use 0.0.0
+try:
+    version_path = Path(__file__).parent / 'VERSION'
```

Comment:
After 8270ec93bfbbff3057dd7d17d1a85394e4b09ade, this should be `Path(__file__) / 'VERSION'`, right? Or else it will be looking for `../VERSION` and it will ALWAYS get `FileNotFoundError`

Reply:
No, the `parent` attr gives the rel path to the parent dir, which in this case would be `pmdarima`.

```python
# from inside the inner dir:
>>> Path("__init__.py").parent / "VERSION"
'./VERSION'
```

So it's getting the '.' prepended basically.

Reply:
Ah, then my previous code was wrong. Good catch!